### PR TITLE
copy: fix Image ReportWriter Option

### PR DESCRIPTION
**Pull request by runcom**
Fix https://github.com/containers/image/pull/102#pullrequestreview-1550771

@mtrmac PTAL

Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

File: copy/copy.go [hunk 1]
```diff
@@ -91,8 +91,9 @@ type Options struct {
...
 // Image copies image from srcRef to destRef, using policyContext to validate source image admissibility.
 func Image(ctx *types.SystemContext, policyContext *signature.PolicyContext, destRef, srcRef types.ImageReference, options *Options) error {
-	if options.ReportWriter == nil {
-		options.ReportWriter = ioutil.Discard
+	reportWriter := options.ReportWriter
+	if reportWriter == nil {
+		reportWriter = ioutil.Discard
 	}
 	writeReport := func(f string, a ...interface{}) {
 		fmt.Fprintf(options.ReportWriter, f, a...)
```
Comment (mtrmac) [inline copy/copy.go]: This needs updating as well. (Or get rid of the function, perhaps,)

Comment (runcom) [inline copy/copy.go]: @mtrmac fixed, sorry not feeling well today and keep missing stuff

**Comment by mtrmac** [comment]
:+1: Thanks!

**Comment by runcom** [comment]
lgtm

**Pull request merged**